profiles/base.yaml

```diff
@@ -537,6 +537,9 @@ safe_room_tip_amount:
 safe_room_empath:
 safe_room_empaths:
 
+# override the room you go to when hunting-buddy ends
```

Comment:
I suggest adding a note that the default if this is not provided is `safe_room`. But it's not a big deal either way.